app/models/bpldc/license.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Bpldc::License < ApplicationRecord
+  validates :label, presence: true
+
+  def self.all_for_api
+    all.map { |rec| rec.as_json.slice('label', 'uri').compact }
```

Comment:
better way to do this is `select('label', 'uri').map { |rec|  rec.as_json.compact }`